### PR TITLE
Update USAGE.md

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -215,18 +218,18 @@ $ brew cask search '/^google.c[a-z]rome$/'
 google-chrome
 ```
 
-## Other Ways to Specify a Cask
+## Other ways to specify a Cask
 
 Most `brew cask` commands can accept a Cask token as an argument. As described above, the token on the command line can take the form of:
 
-* A token as returned by `brew cask search`, _eg_: `google-chrome`.
```
Comment (vitorgalvao) [inline USAGE.md]: We usually like periods at the end of list items.

File: USAGE.md [hunk 2]
```diff
@@ -215,18 +218,18 @@ $ brew cask search '/^google.c[a-z]rome$/'
...
-* A fully-qualified token which includes the Tap, _eg_: `caskroom/fonts/font-symbola`.
+* A token as returned by `brew cask search`, _eg_: `google-chrome`
+* A fully-qualified token which includes the Tap, _eg_: `caskroom/fonts/font-symbola`
 
-`brew cask` also accepts three other forms as arguments
+`brew cask` also accepts three other forms as arguments:
 
-* A path to a Cask file, _eg_: `/usr/local/Library/Taps/caskroom/homebrew-cask/Casks/google-chrome.rb`.
```
Comment (vitorgalvao) [inline USAGE.md]: Same here.

File: USAGE.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# How to Use Homebrew-cask
+# How to use Homebrew-Cask
 
-## Getting Started
+## Getting started
```
Comment (vitorgalvao) [inline USAGE.md]: Same thing about downcasing.

File: USAGE.md [hunk 4]
```diff
@@ -154,7 +157,7 @@ You can tap any of the above with a `brew tap` command:
 $ brew tap <tap_name>
 ```
 
-after which, Casks from the new Tap will be available to `search` or `install` just like Casks from the main Tap. `brew update` will automatically keep your new Tap up to date.
+after which, Casks from the new Tap will be available to `search` or `install` just like Casks from the main Tap. `brew update` will automatically keep your new Tap up-to-date.
```
Comment (adityadalal924) [inline USAGE.md]: The original was correct.

**Comment by miccal** [comment]
All fixed (I think!).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 9d3cb33b0a09cf2039c99877c3bb4b4c25b0876a. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**